### ISSUE TITLE
As a user, I want a beautiful website, so I feel pleasured with bucoli.co

### ISSUE THREAD

**Issue by vjpixel** [memeLab/bucoli.co#59]
Since we're will keep including new features, and it's an image-focused website, it's important to have it well designed and pleasant.

@bruquevedo is going to share the proposal in the comments. _Deadline: Thu 12:00_

@tiagobugarin will implement it. _Deadline: Fri 12:00_

We want to improve the design of About page.

**Comment by vjpixel** [comment]
I propose we add a menu button, like the one in the picture.

![Screenshot from 2020-04-06 10-30-24](https://user-images.githubusercontent.com/1034796/78570494-71534280-77fb-11ea-8c75-c3101cee06d0.png)

**Comment by vjpixel** [comment]
This issue wasn't finished the last sprint. We need to pay attention to finish it so we deliver value to the users.

Please treat this as a new issue, including all necessary information here, so we centralize the discussion. @tiagobugarin @JoshuSantos

**Comment by JoshuSantos** [comment]
> I propose we add a menu button, like the one in the picture.
> 
> ![Screenshot from 2020-04-06 10-30-24](https://user-images.githubusercontent.com/1034796/78570494-71534280-77fb-11ea-8c75-c3101cee06d0.png)

@vjpixel , this is quite a nice menu icon... very minimalist and common ... GitHub uses it as well. 
I can research some others.

My question is:

**Which ITENS will be on the menu?**

"Sobre"
"Galeria"
"Manifesto Bucolico"
"Contato"

Something like that? 
According to the answer, we will have to design the specific pages as well.
Some will have photos or a grid of photos, others will have a form.

It may take longer than 1 sprit because it depends on the number of pages.

What do you think, @tiagobugarin ?

**Comment by vjpixel** [comment]
The items we have now are `Sobre` and `Feedback`. We had both before, but we hide the `Feedback` because we decide to don't use it at that time, so we already have the screen - but maybe it can be customized.

**Comment by tiagobugarin** [comment]
I don't think it would take us too much time to do this. It may take time to make it pleasant and gorgeous. it's the details!!! :)

Also, there is some room with UserReport's interface for adjustments in style but not so much.

**Comment by tiagobugarin** [comment]
the menu is done.
its aesthetics and behavior is nice but imo it still needs some refinements.
the animation may still be too fast at some places and too slow at others. the colors may get a little shift, some more tint, some less...

or we can move to another direction completely... let me know @JoshuSantos @vjpixel @bruquevedo your opinions about the aesthetics and behavior.

**Comment by vjpixel** [comment]
This menu seems to not work with bright images.

**Comment by bruquevedo** [comment]
@tiagobugarin the menu button is technically working and I and I liked the rounded design, but I think that it needs more opacity or some kind of shadow because it's hard to see it

**Comment by tiagobugarin** [comment]
i thought long and hard about this over night and got entangled between not disturbing the picture experience and being noticeable. went down the picture way... may be the sleepness got to my senses and went too far.

@bruquevedo, give another try now and tell me what you think

**Comment by bruquevedo** [comment]
@tiagobugarin for me it's good. It's not disturbing the image and once you take a look in all of it you can find the menu. Plus in the email I gave the path to where it is.

**Comment by tiagobugarin** [comment]
So, @bruquevedo, would you please review this issue?

**Comment by bruquevedo** [comment]
for this sprint I think that everthing is great :)

**Comment by bruquevedo** [comment]
once @vjpixel is still working in the new text #75 and @tiagobugarin is developing some changes to the about page, can I extend my deadline to after you finish your tasks? It'll be easier to me if I can proposal based on this changes

**Comment by vjpixel** [comment]
If you approve the size of my text (600 characters), I think you can go on now. So there's no need to delay.

**Comment by bruquevedo** [comment]
![Proposta-Design-Sobre](https://user-images.githubusercontent.com/63117285/80732560-e1937200-8ae2-11ea-9a38-1920c8f4884d.jpg)

This is my proposal (based in some suggestions Joshu did previously). I think that the font text we are using now is too heavy and I propose to change to the old and know Arial and we can work the weight of some words using bold and italic.
The gray background square (#b5b5b5), a solution we already use, helps to make the text more readable and makes the design lighter.

**Comment by bruquevedo** [comment]
@vjpixel and @tiagobugarin above you can see my proposal

**Comment by vjpixel** [comment]
I like more @tiagobugarin approach (that is in the development environment now).

**Comment by vjpixel** [comment]
I suggest removing bold from the first fugere urbem (the one at the beginning of the text).

**Comment by vjpixel** [comment]
Fugere urbem é uma expressão em latim que significa _fugir da cidade_. Originalmente usada pelo escritor latino Horácio, foi adotada como lema pela literatura árcade para simbolizar o poeta que se desloca da vida agitada da cidade e vai para a calma zona rural.

A proposta de **bucoli.co** é aproveitar alguns instantes do enorme tempo que passamos olhando para telas para fugir da poluição visual urbana. A cada semana, há uma nova imagem de uma paisagem bucólica, de artistas que convidamos. É uma oportunidade de fugir um pouco do estresse do confinamento e _voltar às origens_.

**Fugere urbem!**

**Comment by tiagobugarin** [comment]
changes applyed to alpha.

**Comment by vjpixel** [comment]
It seems good to me.
